setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
         "eth-abi>=2.1.1,<3",
         "eth-typing>=2.2.1,<3",
         "eth-utils>=1.8.4,<2",
-        "eth-hash[pycryptodome]>=0.2.0,<0.3",
+        "eth-hash[pycryptodome]>=0.3.1,<0.4", #make version range comaptible with eth-utils requirements
```

Comment:
That comment (without the typo in compatible) should be the commit message and not a comment here
```suggestion
        "eth-hash[pycryptodome]>=0.3.1,<0.4",
```